src/components/AssetsTableContent.jsx

```diff
@@ -60,7 +60,7 @@ const AssetsTableContent = (props) => {
               serial_number: asset.serial_number || '-',
               model_number: asset.model_number || '-',
               assignee: (asset.assigned_to && asset.assigned_to.email)
-                || (asset.assigned_to && `${asset.assigned_to.full_name}`)
+                || (asset.assigned_to && asset.assigned_to.full_name)
                 || '-'
```

Comment:
Just a small bug I noticed while working on this task. I was hoping I could sneak it in this PR :)

Reply:
@rotimi-babalola is it related to https://www.pivotaltracker.com/story/show/162481628?

Reply:
Yes it is @egichuri I wasn't aware there was a story for this. I'll remove this fix from this PR and create a branch for that story with this fix. Thanks